.github/workflows/math_coding_ci_dev_workflow.yml

```diff
@@ -20,7 +20,7 @@ on:
   push:
     branches:
       - main
-      - development
+      # - development
```

Comment:
Commented this out so these other workflows won't run when PRs are open. Open to suggestions on if I should just delete all the workflows that are part of the sample.

Reply:
I think moving the samples to a different folder, and keeping the folder that runs the workflow to non-sample ones makes sense to me. 

I like having the 'development' and 'main' branch triggers, as it'll enable teams to run integration tests using a different size of datasets. small / quick datasets for integration test against 'development' branch, and larger / more rigorous datasets for integration tests in `main`.

Reply:
- i vote to delete them. your readme can reference that the template repo has actual examples.